tests/test_bpo.py

```diff
@@ -10,6 +10,8 @@ class FakeGH:
 
     def __init__(self, *, getitem=None):
         self._getitem_return = getitem
+        self.patchurl = None
```

Comment:
Underscores have been used in the rest of the test code, so `patch_url` and `patch_data` for consistency.